core/server/src/main/java/alluxio/master/AlluxioMasterRestServiceHandler.java

```diff
@@ -134,10 +139,10 @@ public AlluxioMasterInfo call() throws Exception {
   }
 
   /**
-   * @summary get the configuration map, the keys are ordered alphabetically.
    * @return the response object
-   * @deprecated since version 1.4 and will be removed in version 2.0
+   * @summary get the configuration map, the keys are ordered alphabetically.
```

Comment:
why is summary after return? should javadoc for other REST API handler files be changed accordingly